### ISSUE TITLE
don't see Database menu in Tools

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#1638]
Just updated to latest Github

Developer mode enabled

Cache emptied, no Database in Tools menu

![image](https://cloud.githubusercontent.com/assets/5429548/15145749/bcf8f364-16b7-11e6-899d-964e0c169876.png)

I did the update via this functionality before, waiting for message of database upgrade - this message is not there too. I did the same thing on localhost and it worked. 

Any ideas?

**Comment by Jimmi08** [comment]
I tried disabled and enabled debug mode
![image](https://cloud.githubusercontent.com/assets/5429548/15145886/62db28c4-16b8-11e6-956b-970666450532.png)

And in Firefox I see the debug mode is on
![image](https://cloud.githubusercontent.com/assets/5429548/15145895/727e45a4-16b8-11e6-92d2-76e4e0dd8dba.png)

**Comment by CaMer0n** [comment]
@Moc Is it the same for you?

**Comment by Moc** [comment]
@CaMer0n Actually no :D Sorry was still in the process of testing this but I already labelled it as a bug.

**Comment by CaMer0n** [comment]
@Jimmi08 Your user_perms = 0 ?

**Comment by Jimmi08** [comment]
@CaMer0n It's my account, main admin, I am home now, I can compare things now.

**Comment by Jimmi08** [comment]
OK, this is weird. I was main admin, I updated site and my permissions are now:
![image](https://cloud.githubusercontent.com/assets/5429548/15156798/fba5ceb2-16e8-11e6-9c05-ed6736bb446f.png)
I remember that I changed my password, nothing more besides update.

Then I changed value in database to 0 and tools are back. Really don't know what happened because it's not so easy to tick all permissions manually.

But I noticed something else. As main admin I have less icons at dashboard that admin with all perms checked. Is it purpose?
